src/main.rs

```diff
@@ -238,6 +239,7 @@ fn real_main(options: Args, config: &mut Config) -> CliResult {
                             s.to_owned()
                         }
                     }
+                    GitReference::DefaultBranch => String::from("${AUTOREV}"),
```

Comment:
can't we remove the check for `master` above now that we have `DefaultBranch` in the enum?

Reply:
In 99% of cases there probably wouldn't be a difference, and we could remove that. But if someone had specified `branch = 'master'` then removing this check could change the behaviour they expect. So I'd rather leave it in unless there is a reason to remove it.